packages/vue-components/src/Navbar.vue

```diff
@@ -305,6 +361,11 @@ export default {
     }
 
     /* Navbar link highlight for current page */
+    .navbar .navbar-nav >>> .current:not(.dropdown),
+    .navbar .navbar-nav >>> .dropdown.current {
+        background: none;
```

Comment:
This is so that when a link in the dropdown is highlighted, the dropdown itself will be highlighted as well.

![image](https://user-images.githubusercontent.com/47470981/145837611-5cf0e207-8db5-4341-9823-097649299ea8.png)